apis/debug/state.v2.yaml

```diff
@@ -0,0 +1,61 @@
+get:
+  operationId: "getStateV2"
+  summary: "Get full BeaconState object"
+  description: |
+     Returns full BeaconState object for given stateId.
+     Depending on `Accept` header it can be returned either as json or as bytes serialized by SSZ
+  tags:
+    - Debug
+  parameters:
+    - name: state_id
```

Comment:
param can be reused like here: https://github.com/ethereum/eth2.0-APIs/blob/master/apis/beacon/states/committee.yaml#L8

Reply:
this was literally a copy of state.yaml, should i update that too?

Reply:
Those things can sometime slip trough cracks, tnx for fixing 😄